extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -57,28 +60,24 @@ class MonetizationModuleHelper extends WikiaModel {
 
 	/**
 	 * Show the Module only on File pages, Article pages, and Main pages
+	 * @param Title $title
 	 * @return boolean
 	 */
-	public static function canShowModule() {
-		wfProfileIn( __METHOD__ );
-
+	public static function canShowModule( Title $title ) {
 		$app = F::app();
 		$status = false;
 		$showableNameSpaces = array_merge( $app->wg->ContentNamespaces, [ NS_FILE ] );
 		if ( !WikiaPageType::isCorporatePage()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (6) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)